doc/languages-frameworks/haskell.md

```diff
@@ -334,16 +334,30 @@ navigate there.
 
 Finally, you can run
 ```shell
-hoogle server -p 8080
+hoogle server --local -p 8080
 ```
 and navigate to http://localhost:8080/ for your own local
-[Hoogle](https://www.haskell.org/hoogle/). Note, however, that Firefox and
-possibly other browsers disallow navigation from `http:` to `file:` URIs for
-security reasons, which might be quite an inconvenience. Since version 5 hoogle
-server has a `--local` flag that solves the problem. For older versions see
+[Hoogle](https://www.haskell.org/hoogle/). The `--local` flag makes the hoogle
+server serve files from your nix store over http, without the flag it will use
+`file:\\` URIs. Note, however, that Firefox and possibly other browsers
```

Comment:
You accidentally flipped some slashes here

Reply:
I felt like the backslashes were starting to get outnumbered ; )